spec/rest/auth.spec.js

```diff
@@ -0,0 +1,58 @@
+"use strict";
+
+define(['ably', 'shared_helper'], function(Ably, helper) {
+  beforeAll(helper.setupApp);
+  afterAll(helper.tearDownApp);
+
+  describe('REST Auth', function() {
+    var currentTime; // set by first test
+
+    describe('Ably.Rest#time', function() {
+      it('obtains the server time', function(done) {
+        helper.AblyRest({ key: 0 }).time(function(err, time) {
+          if(err) { throw err; }
+          var timeOffset = time - Date.now();
+          expect(Math.abs(timeOffset)).toBeLessThan(5000);
+          currentTime = Math.floor(time/1000);
```

Comment:
Yes @kouno, this is actually a set up task, and it's copied & pasted from the existing tests.